src/table/components/TableBodyWrapper.tsx

```diff
@@ -38,7 +39,14 @@ function TableBodyWrapper({
   );
   const bodyCellStyle = useMemo(() => getBodyCellStyle(layout, theme), [layout, theme]);
   const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
-  const cellStyle = { color: bodyCellStyle.color, backgroundColor: theme.table.backgroundColor };
+  const getCellStyle = (columnIndex: number) => {
+    return {
+      color: bodyCellStyle.color,
+      backgroundColor: theme.table.backgroundColor,
+      opacity: engagedColumn !== undefined && engagedColumn !== columnIndex ? '50%' : null,
+    };
+  };
```

Comment:
```suggestion
const getCellStyle = (columnIndex: number) => ({
    color: bodyCellStyle.color,
    backgroundColor: theme.table.backgroundColor,
    opacity: engagedColumn !== undefined && engagedColumn !== columnIndex ? '50%' : null,
  });
```
Not big deal, just looks cleaner.

Reply:
sure